packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -17,6 +17,12 @@ interface IDependencySet {
     function isInDependencySet(uint256 _chainId) external view returns (bool);
 }
 
+/// @notice Thrown when the caller is not DEPOSITOR_ACCOUNT when calling `setInteropStart()`
+error OnlyDepositorAccount();
```

Comment:
maybe we want to use the same error as here: https://github.com/ethereum-optimism/optimism/blob/b047e1fd3b16d76c820d6e8cfc3cac4ede09d847/packages/contracts-bedrock/src/L2/L1Block.sol#L140-L143

ie `error NotDepositor` just to keep it consistent

You can see in the gas price oracle we are still on string errors rather than custom errors
https://github.com/ethereum-optimism/optimism/blob/b047e1fd3b16d76c820d6e8cfc3cac4ede09d847/packages/contracts-bedrock/src/L2/GasPriceOracle.sol#L81

It would be ideal to try to use the same set of errors when possible rather than introducing new ones for every case